src/libMessage/ZilliqaMessage.proto

```diff
@@ -439,3 +439,31 @@ message ConsensusCommitFailure
     required ConsensusInfo consensusinfo = 1;
     required ByteArray signature         = 2;
 }
+
+message MicroBlockInfo
```

Comment:
Please make this a nested message inside `LookupGetMicroBlockFromLookup`

Reply:
@ansnunez I needed to make use of this type (in declaration) so brought it outside.